### PR TITLE
ENH separate out long-term migrations for the status page

### PR DESCRIPTION
**Pull request by beckermr**
This PR separates out the long-term migrations.

### CODE REVIEW

File: conda_forge_tick/status_report.py [hunk 1]
```diff
@@ -323,8 +341,17 @@ def main(args: Any = None) -> None:
...
+
+    with open("./status/longterm_status.json", "w") as f:
+        json.dump(regular_status, f, sort_keys=True)
+
+    total_status = {}
+    total_status.update(regular_status)
+    total_status.update(longterm_status)
     with open("./status/total_status.json", "w") as f:
-        json.dump(total_status, f, sort_keys=True)
+        json.dump(regular_status, f, sort_keys=True)
```
Comment (beckermr) [inline conda_forge_tick/status_report.py:354]:
```suggestion
        json.dump(total_status, f, sort_keys=True)
```

File: conda_forge_tick/status_report.py [hunk 2]
```diff
@@ -323,8 +341,17 @@ def main(args: Any = None) -> None:
     print(" ", flush=True)
 
     print("writing data", flush=True)
+    with open("./status/regular_status.json", "w") as f:
+        json.dump(regular_status, f, sort_keys=True)
+
+    with open("./status/longterm_status.json", "w") as f:
+        json.dump(regular_status, f, sort_keys=True)
```
Comment (beckermr) [inline conda_forge_tick/status_report.py:348]:
```suggestion
        json.dump(longterm_status, f, sort_keys=True)
```

**Comment by ocefpaf** [comment]
Not important but how difficulty would it be to get a snapshot of the status page with this PR?

**Comment by beckermr** [comment]
This PR still writes total_status, so we can merge it and everything will work as it does now until we change the status page. 

I have not changed the status page yet. Once the bot is back online, I will put in that PR. It needs the new status data being generated here.

**Pull request merged**